### PR TITLE
prow: fuzzy search should not include jobs that do not belong to a repo

### PR DESCRIPTION
**Pull request by qhuynh96**

### CODE REVIEW

File: prow/cmd/deck/static/script.js [hunk 1]
```diff
@@ -433,7 +437,11 @@ function redraw() {
         emitted++;
 
         var r = document.createElement("tr");
-        r.appendChild(stateCell(build.state));
+        if (build.state === "" ) {
+            r.appendChild(createTextCell(""));
+        } else {
+            r.appendChild(stateCell(build.state));
```
Comment (stevekuznetsov) [inline prow/cmd/deck/static/script.js]: Or update `stateCell()` to handle the `build.state === ""` case?

Comment (qhuynh96) [reply prow/cmd/deck/static/script.js]: done

**Comment by qhuynh96** [comment]
@stevekuznetsov friendly poke. @BenTheElder is busy this week so can you have a look?

File: prow/cmd/deck/static/script.js [hunk 2]
```diff
@@ -30,8 +30,8 @@ function optionsForRepo(repo) {
         var build = allBuilds[i];
         opts.types[build.type] = true;
         opts.repos[build.repo] = true;
-        if (!repo || repo === build.repo) {
-            opts.jobs[build.job] = true;
+      if (!repo || repo === build.repo) {
```
Comment (stevekuznetsov) [inline prow/cmd/deck/static/script.js]: bad whitespace changes?

Comment (qhuynh96) [reply prow/cmd/deck/static/script.js]: done

**Comment by stevekuznetsov** [comment]
Changes here
/lgtm

**Pull request merged**